test/unit/specs/components/DataCollector/index.spec.js

```diff
@@ -60,7 +60,8 @@ describe("Event Command", () => {
       expect(event.setUserData).toHaveBeenCalledWith(data);
       expect(eventManager.sendEvent).toHaveBeenCalledWith(event, {
         renderDecisions: true,
```

Comment:
Reevert

Reply:
I have reverted the changes related to the new option from the API and the sandbox, but as @jfkhoury mentioned in a previous comment, we need these to be present in tests, in order to confirm that both the top-level `decisionScopes` and the new `personalization.decisionScopes` options are working as expected.